### ISSUE TITLE
Q&V: su ceramics

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#2381]

**Comment by urapadmin** [comment]
### kiosk 1.5.39.1
- after #2728 is solved the custom ui definition for pasu should show a proper view if you navigate to a ceramic fragment like e.g. '#038-006'. But please test a bunch of them! With images!

**Comment by urapadmin** [comment]
- see also #2382 for the ceramic fragments

**Comment by urapadmin** [comment]
needs to be tested on local PASU, too!

**Comment by luizaogs** [comment]
Somewhat related but not opening a ticket yet because I don't know if it's a problem: if a project doesn't use lots, should the field show up in Q&V? I just noticed a blank lot field when testing this in PASU: 

<img width="1512" alt="Screenshot 2024-05-27 at 2 48 42 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/e32b5625-974f-41dd-b3a4-c886476348f4">

**Comment by luizaogs** [comment]
Weight and quantity fields look a bit wacky: 

<img width="408" alt="Screenshot 2024-05-27 at 2 49 45 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/391d12c4-63a1-45b3-bbd7-1ce21d023672">

**Comment by luizaogs** [comment]
You're also currently missing the total count and weight, visible in the app like this: 

<img width="1017" alt="Screenshot 2024-05-27 at 2 53 35 PM" src="https://github.com/arch-kiosk/arch-kiosk-office/assets/67970402/5a16b1e1-4a84-4b47-9331-e389b95bad9d">

**Comment by luizaogs** [comment]
I believe the rest is ok.

**Comment by urapadmin** [comment]
> Somewhat related but not opening a ticket yet because I don't know if it's a problem: if a project doesn't use lots, should the field show up in Q&V? I just noticed a blank lot field when testing this in PASU:


worth a ticket

**Comment by urapadmin** [comment]
> Weight and quantity fields look a bit wacky:
> <img alt="Screenshot 2024-05-27 at 2 49 45 PM" width="408" src="https://private-user-images.githubusercontent.com/67970402/334216801-391d12c4-63a1-45b3-bbd7-1ce21d023672.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NTAzNzMsIm5iZiI6MTcxNjg1MDA3MywicGF0aCI6Ii82Nzk3MDQwMi8zMzQyMTY4MDEtMzkxZDEyYzQtNjNhMS00NWIzLWJiZDctMWNlMjFkMDIzNjcyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDIyNDc1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIzMTk3OWFmYTY4NzUyOTRlM2Y4MjEyZjM1MGI1NTc3OGZiM2ZhMjc3OTIxYzc2M2U2YjhmMjA2YjBkNmU2NTgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Nx9M9wxv8fuMy0mcQ2z0ZAPEjMTrW5X7rU_dSrZOZQ0">

mainly because of #2727

**Comment by urapadmin** [comment]
> You're also currently missing the total count and weight, visible in the app like this:
> <img alt="Screenshot 2024-05-27 at 2 53 35 PM" width="1017" src="https://private-user-images.githubusercontent.com/67970402/334217186-5a16b1e1-4a84-4b47-9331-e389b95bad9d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NTAzNzMsIm5iZiI6MTcxNjg1MDA3MywicGF0aCI6Ii82Nzk3MDQwMi8zMzQyMTcxODYtNWExNmIxZTEtNGE4NC00YjQ3LTkzMzEtZTM4OWI5NWJhZDlkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI3VDIyNDc1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUzYmY0OWFkYTQzZWEzMzM2ZDkwZDAxNmEzMzA3ZTYwMTY4MWFkYmQ4ODI4ZjM4ZGU5ZjUxZmIxMmZlYWY1NTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-OpywLdJXX4YB6EE1uKdp3H0s3LqCr1mDsFZoscSxV8">

Those are live calculations. Can't do that in the current state of View. But the totals should be fields in the collected material section.

**Comment by luizaogs** [comment]
> > Somewhat related but not opening a ticket yet because I don't know if it's a problem: if a project doesn't use lots, should the field show up in Q&V? I just noticed a blank lot field when testing this in PASU:
> 
> worth a ticket

Done.

**Comment by urapadmin** [comment]
#2727 is out, so this look better now. #2735 should be solved, too.
Anything I skipped here?

**Comment by luizaogs** [comment]
No, looks fine now. Over to @lbestock for the local test.

**Comment by lbestock** [comment]
This looks good, and I don't have any trouble getting to quantity and weight totals from the CM. And I get the fragments right below, just as I would like. But...I do miss the graphs. We have them in Q&V for materials in general, and we have them in FM for preprocessing, but we don't have them in Q&V for preprocessing.

**Comment by lbestock** [comment]
All the numbers I have checked are correct. So #2738, which is a terminology issue, and my above desire for pie charts. Not sure if those mean the ticket should stay open or not.

**Comment by urapadmin** [comment]
> This looks good, and I don't have any trouble getting to quantity and weight totals from the CM. And I get the fragments right below, just as I would like. But...I do miss the graphs. We have them in Q&V for materials in general, and we have them in FM for preprocessing, but we don't have them in Q&V for preprocessing.



> This looks good, and I don't have any trouble getting to quantity and weight totals from the CM. And I get the fragments right below, just as I would like. But...I do miss the graphs. We have them in Q&V for materials in general, and we have them in FM for preprocessing, but we don't have them in Q&V for preprocessing.

That would be a new query (they won't appear in a view), which is feasible but please make it a new ticket.

**Comment by urapadmin** [comment]
so this can be closed then.

**Comment by luizaogs** [comment]
> > This looks good, and I don't have any trouble getting to quantity and weight totals from the CM. And I get the fragments right below, just as I would like. But...I do miss the graphs. We have them in Q&V for materials in general, and we have them in FM for preprocessing, but we don't have them in Q&V for preprocessing.
> 
> > This looks good, and I don't have any trouble getting to quantity and weight totals from the CM. And I get the fragments right below, just as I would like. But...I do miss the graphs. We have them in Q&V for materials in general, and we have them in FM for preprocessing, but we don't have them in Q&V for preprocessing.
> 
> That would be a new query (they won't appear in a view), which is feasible but please make it a new ticket.

I opened this for Laurel.
